source/extensions/transport_sockets/alts/tsi_socket.cc

```diff
@@ -133,9 +133,14 @@ Network::PostIoAction TsiSocket::doHandshakeNextDone(NextResultPtr&& next_result
     // returns TSI_OK assuming there is no fatal error. Asserting OK.
     tsi_zero_copy_grpc_protector* frame_protector;
     grpc_core::ExecCtx exec_ctx;
-    status = tsi_handshaker_result_create_zero_copy_grpc_protector(handshaker_result, nullptr,
+    status = tsi_handshaker_result_create_zero_copy_grpc_protector(handshaker_result, &default_max_frame_size_,
```

Comment:
The formatter really wants you to do a few changes, please look at:

https://dev.azure.com/cncf/envoy/_build/results?buildId=68287&view=logs&j=55bc2ab9-36c1-5ab8-58a6-8c258728eeff&t=33b57bf9-ef13-58c8-bc2b-f804aeb16d39

Reply:
Done.